expyfun/stimuli/_tracker.py

```diff
@@ -820,9 +820,9 @@ def respond(self, correct):
         if self.stopped:
             self._callback(
                 'dealer_%i_stop' % self._dealer_id, json.dumps(dict(
-                    tracker_position=[int(s) for s in self._tracker_history],
-                    responses=[int(s) for s in self._response_history],
-                    x=[int(s) for s in self._x_history])))
+                    tracker_history=[int(s) for s in self._tracker_history],
+                    response_history=[int(s) for s in self._response_history],
+                    x_history=[float(s) for s in self._x_history])))
```

Comment:
Using `float(s)` now makes it unhappy for `Tracker_Binom` where `x_current=None`